UnityProject/Assets/Scripts/Core/ITrackableType.cs

```diff
@@ -46,6 +46,42 @@ public static List<T> GetAllNearbyTypesToTarget(GameObject target, float maximum
 			return components;
 		}
 
+		public static List<T> GetAllNearbyTypesToLocation(Vector3 target, float maximumDistance, bool bypassInventories = true)
+		{
+			if (Instances == null || Instances.Count == 0)
+			{
+				Loggy.Log($"No elements found for Type {nameof(T)}, are you sure you have ITrackableType<T> added to your class?");
+				return null;
+			}
+#if UNITY_EDITOR
+			var stopwatch = new Stopwatch();
+			stopwatch.Start();
+#endif
+			List<T> components = new List<T>();
+			foreach (var stationObject in Instances)
```

Comment:
Instances?

Reply:
This does the same thing as `GetAllNearbyTypesToTarget` but uses a Vector3 instead of GameObject as input.

Reply:
maybe ``positions`` would be more suitable then

Reply:
No, that is a misleading name because of this `public static HashSet<T> Instances { get; } = new HashSet<T>();`

Please read the full script before suggesting anything.